lib/logger.js

```diff
@@ -3,32 +3,37 @@
 const colors = require('colors');
 const stdout = require('mute-stdout');
 
+let logLevel = null;
+
 class Logger {
```

Comment:
This module exposes the class `Logger` via `module.exports = Logger;`. This means every time `Logger()` is called a new instance is created. The only change you made is setting a `logLevel` shared between all loggers. Instead you should use or wrap it into a singleton pattern:
```javascript
let instance;

function getLogger (level) {
  if (!instance) {
    instance = new Logger(level);
  }
  return instance;
}

module.exports = getLogger;
```

This way there will always be only one `Logger` instance. Now the code does what it says. And as a bonus it also reduces memory use as it only needs to keep track of one instance.